### DISCUSSION TITLE
How to ask a good question.

### DISCUSSION THREAD

**Discussion by joaander** [glotzerlab/freud#1186] [Announcements]
If you want to quickly get a good answer to your question, you *must* ask a **good question**. Here are a few resources to help:
* [How do I ask a good question?](https://stackoverflow.com/help/how-to-ask)
* [Writing the perfect question](https://codeblog.jonskeet.uk/2010/08/29/writing-the-perfect-question/)
* [How to create a minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)

**Comment by Gareth-Bellinger** [comment]
Hi there, 
I was hoping to ask a new question in the discussion section but I am unable to see a 'New Discussion' button. I can see this button on other github pages. Apologies if this is an issue on my end, as I am a new github user.

**Reply by joaander** [reply]
Try it now.

**Reply by Gareth-Bellinger** [reply]
I see it now! Thank you.
